packages/api-contract/src/base/Code.ts

```diff
@@ -34,6 +34,16 @@ export class CodeSubmittableResult<ApiType extends ApiTypes> extends Submittable
   }
 }
 
+function isRiscV (bytes: unknown) {
```

Comment:
```suggestion
function isRiscV (bytes: unknown): bytes is Uint8Array {
```

This just gets the typings right as well. Every now and again TS has trouble with "auto" (although here it _should_ certainly detect the type as `Uint8Array`), but either way it is preferable being more explicit)